src/pages/home/report/ReportActionItem.js

```diff
@@ -97,8 +97,10 @@ class ReportActionItem extends Component {
      * @param {string} [selection] - A copy text.
      */
     showPopover(event, selection) {
-        // Block menu on the message being Edited or is already deleted
-        if (this.props.draftMessage || ReportUtils.isDeletedAction(this.props.action)) {
+        const className = event.target.className || null;
+
+        // Block menu on the message being Edited or is already deleted or event.target is scrollbar
+        if (this.props.draftMessage || ReportUtils.isDeletedAction(this.props.action) || (className && className.includes('WebkitOverflowScrolling'))) {
```

Comment:
This is not a good approach no hacks. className does not have to do anything with RN. Let's try to find a proper solution so the problem is in react way. The best approaches would be:
1. Either prevent the event from bubbling so it never reached menu target. 
2. Use some code to detect that the user is interacting and does prevent a popover. It could work for any kind of interaction that we define. But for this issue just scrolling the child.